src/middlewares/validation.middleware.ts

```diff
@@ -101,29 +119,35 @@ export async function validationMiddleware(options: ValidationMiddlewareOptions)
   return async function (req: Request, _: Response, next: NextFunction) {
     // validate request body
     try {
-      await validateRequestBody(validate, req.body);
+      if (validate) {
+        await validateRequestBody(validate, req.body);
+      } else {
+        await validateRequestParameters(req.params, next);
```

Comment:
You can have requestBody and parameters as well, so you should try to validate request body and/or parameters :)